tests/keras_tests/tpc_keras.py

```diff
@@ -16,8 +16,9 @@
 from keras.engine.input_layer import InputLayer
 import tensorflow as tf
 import model_compression_toolkit as mct
-from model_compression_toolkit.core.tpc_models.keras_tp_models.keras_default import generate_keras_default_tpc
+
```

Comment:
Need test for every Hardware model for each of is versions

Reply:
Done